### ISSUE TITLE
Karma tests break

### ISSUE THREAD

**Issue by kayvanbree** [kyubisation/angular-server-side-configuration#4]
Running my server side configuration through Jenkins causes `npm test` to fail on redeclaring process:

```
ERROR in node_modules/@types/node/index.d.ts(97,13): error TS2451: Cannot redeclare block-scoped variable 'process'.
node_modules/angular-server-side-configuration/process.d.ts(1,13): error TS2451: Cannot redeclare block-scoped variable 'process'.
```

I'm uncommenting my testing stage now to see if it all works, but this should be fixed somehow. I don't really know where to start looking tough. Did you get Karma test to work?

**Comment by kyubisation** [comment]
The problem was an incompatible process declaration in process.d.ts.
I have fixed this and released version 0.2.1.

**Comment by kayvanbree** [comment]
Awesome!
